src/util/animation/AnimationFrameProvider.ts

```diff
@@ -1,28 +1,9 @@
-import { getEnv } from '../../env';
+import { getWindow } from '../../env';
 
-let _requestAnimFrame: AnimationFrameProvider['requestAnimationFrame'];
-let _cancelAnimFrame: AnimationFrameProvider['cancelAnimationFrame'];
-
-/**
- * requestAnimationFrame polyfill based on http://paulirish.com/2011/requestanimationframe-for-smart-animating/
- * In order to get a precise start time, `requestAnimFrame` should be called as an entry into the method
- * @param {Function} callback Callback to invoke
- */
 export function requestAnimFrame(callback: FrameRequestCallback): number {
-  if (!_requestAnimFrame) {
-    _requestAnimFrame =
-      getEnv().window.requestAnimationFrame ||
-      function requestAnimationFramePolyfill(callback: FrameRequestCallback) {
-        return getEnv().window.setTimeout(callback, 1000 / 60);
-      };
```

Comment:
I am actually for having the polyfill and eventually dumping jsdom

Reply:
dumping JSDOM for now gives us zero benefits, if we find a different backend renderer that does not need it great.
Fabric once had different code for node and web and i don't want to go back there.

Reply:
I am backing off this, but 

> Fabric once had different code for node and web and i don't want to go back there.

That is the exact point of Env, a minimal diff in a common exposed interface.
That is why I think one day JSDOM can fade away.
But is not important.